### DISCUSSION TITLE
Will this work with StereoTool & RadioBoss

### DISCUSSION THREAD

**Discussion by paul19920801** [apohl79/audiogridder#891] [Help]
Hi all. 

I run an online radio station as like a little hobbby. 

My radio station is run in the cloud, and is connected to a VPN, so that I can access all my machines as if it were on a LAN. There is a bit of latency, but not all that much. 

Unfortunately the CPU on the broadcasting machine is inadequate to run stereo tool.  This made me see if I could find a way to somehow get Stereo tool to process the sound remotely on a separate machine, then feed it back to Radioboss. I would still get Stereo tool's amazing sound, except without the buffering. 

After doing some searching, I came across Audiogridder, but after installing both plugin & server I am unable to get it to work. The server doesn't detect the Stereo tool DLL file. I replaced the DSP plugin I had with VST to see if would show up in plugin settings - atill nothing, and when I installed the plugin side on Radioboss it doesn't see it....

Is what I am trying to achieve possible?

Thanks in advance

**Comment by emack999** [comment]
Did you download the [64 bit plugin](https://www.stereotool.com/download/vst_stereo_tool_64.dll)?  Is it installed in a directory that AG is looking to find plugins?

**Reply by paul19920801** [reply]
Yes I have - I specified the custom folder on the server, where the plugin in is located. I then proceed to available settings to see if it "Sees" the plugin, and the available plugin page is blank.  With no plugins listed. 

Here is a screenshot of my config. 

Any idea what I am doing wrong?
![Screenshot](https://user-images.githubusercontent.com/90729302/150651343-c8cb5d98-5b66-4875-aae1-f4d70ed2706f.JPG)

**Reply by emack999** [reply]
I'm not having a problem with AG finding that plugin, so perhaps choose a different folder for your vst.  I suggest using C:\Program Files\VSTPlugins instead of the custom directory you have given.

![1111](https://user-images.githubusercontent.com/81657685/150651566-5843b069-78a0-455f-9fb4-d6be8dd0aeae.jpg)

**Reply by paul19920801** [reply]
Will give it a try. Thanks 
Is the plugin meant to be installed on the server or the workstation with the audio program installed? (RadioBoss)

**Reply by emack999** [reply]
Plug-ins go on the server. You access them on the client machine through AG.

**Comment by paul19920801** [comment]
Will Audiogridder work with x86 plugins by any chance? It seems Radioboss (or at least my version is only x86), and doesn't seem to recognise the 64 bit VST plugins.

**Reply by emack999** [reply]
No support for 32 bit plugins.  You can get something like [jBridge](https://jstuff.wordpress.com/jbridge/) and bridge your 32 bit into 64 bit plugins;

**Comment by paul19920801** [comment]
I have tried jBridge, and can confirm the devices are talking, a small bit of lagging but will address that. Now how does one open the Stereo Tool plugin itself so that I can load the relevant audio settings.

**Reply by emack999** [reply]
On the client machine, inside RadioBoss, open the AG plugin. Once the AG plugin window is open, select the '+' and choose the StereoTool plugin.  The interface for that plugin should load.
